electron_app/src/windows/upgradePlus.js

```diff
@@ -0,0 +1,29 @@
+const { BrowserWindow } = require('electron');
+
+const openUpgradeToPlusWindow = token => {
+  try {
+    const workerWin = new BrowserWindow({
+      width: 400,
+      height: 700,
+      transparent: false,
+      center: true,
+      show: true,
+      modal: false,
+      webPreferences: {
+        nodeIntegration: true
+      }
+    });
+    workerWin.loadURL(
+      `https://admin.criptext.com/?#/account/billing?token=${token}`
```

Comment:
I don't know if Daniel tell you but you can add a param called lang=en where lang is the language of the billing page ['en', 'es', 'ru', 'de', 'fr']